map-downloader-android/app/src/main/AndroidManifest.xml

```diff
@@ -38,12 +38,12 @@
         <service
             android:name="com.here.android.mpa.service.MapService"
             android:label="HereMapService"
```

Comment:
If we want to use an isolated-cache-friendly MapService, shouldn't we also replace the label? https://developer.here.com/mobile-sdks/documentation/android-premium/topics/map-service.html